### PR TITLE
Backport439 sle15sp4 #2381 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

SLE 15 SP4 is base.

https://github.com/SUSE/spacewalk/issues/22684

# Target branches

Which documentation version does this PR apply to?
- [ ] master (Default)
- [ ] manager-4.3

# Links
- Related development PR #<insert PR link, if any>
- Fixes #<insert spacewalk issue, if any>

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,9 +4,11 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- SUSE Manager 4.3 is based on SUSE Linux Enterprise 15 SP5; changed
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- SUSE Manager 4.3 is based on SUSE Linux Enterprise 15 SP4; changed
```

File: modules/installation-and-upgrade/pages/proxy-registration.adoc [hunk 2]
```diff
@@ -33,10 +33,10 @@ image::proxy-key.png[]
...
 . From the [guimenu]``Child Channels`` listing select the recommended channels by clicking the ``include recommended`` icon:
 +
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-SP4-Pool
+* SLE-Module-Basesystem15-SP4-Updates
+* SLE-Module-Server-Applications15-SP4-Pool
+* SLE-Module-Server-Applications15-SP4-Updates
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-registration.adoc:39]:
```suggestion
* SLE-Module-Basesystem15-{sp-base}-Pool
* SLE-Module-Basesystem15-{sp-base}-Updates
* SLE-Module-Server-Applications15-{sp-base}-Pool
* SLE-Module-Server-Applications15-{sp-base}-Updates
```

File: modules/quickstart/pages/quickstart-install-suma-proxy.adoc [hunk 3]
```diff
@@ -61,10 +61,10 @@ image::proxy-key.png[]
...
 . From the [guimenu]``Child Channels`` listing select the recommended channels by clicking the ``include recommended`` icon:
 +
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-SP4-Pool
+* SLE-Module-Basesystem15-SP4-Updates
+* SLE-Module-Server-Applications15-SP4-Pool
+* SLE-Module-Server-Applications15-SP4-Updates
```
Comment (keichwa) [inline modules/quickstart/pages/quickstart-install-suma-proxy.adoc:67]:
```suggestion
* SLE-Module-Basesystem15-{sp-base}-Pool
* SLE-Module-Basesystem15-{sp-base}-Updates
* SLE-Module-Server-Applications15-{sp-base}-Pool
* SLE-Module-Server-Applications15-{sp-base}-Updates
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]

File: modules/installation-and-upgrade/pages/install-vm.adoc [hunk 4]
```diff
@@ -115,10 +115,10 @@ SUSEConnect -e <EMAIL_ADDRESS> -r <SUSE_MANAGER_CODE> \
 +
 
 ----
-SUSEConnect -p sle-module-basesystem/15.5/x86_64
-SUSEConnect -p sle-module-server-applications/15.5/x86_64
-SUSEConnect -p sle-module-web-scripting/15.5/x86_64
-SUSEConnect -p sle-module-suse-manager-server/<productnumber>/x86_64
```
Comment (deneb-alpha) [inline modules/installation-and-upgrade/pages/install-vm.adoc:121]: on the proxy string for suseconnect, I don't know why you removed the productnumber and the arch.

looking at parameters.yml the variable could be
smrproductnumber

File: modules/installation-and-upgrade/pages/install-vm.adoc [hunk 5]
```diff
@@ -115,10 +115,10 @@ SUSEConnect -e <EMAIL_ADDRESS> -r <SUSE_MANAGER_CODE> \
...
+SUSEConnect -p sle-module-basesystem/15.4/x86_64
+SUSEConnect -p sle-module-server-applications/15.4/x86_64
+SUSEConnect -p sle-module-web-scripting/15.4/x86_64
+SUSEConnect -p sle-module-suse-manager-server/
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/install-vm.adoc:121]:
```suggestion
SUSEConnect -p sle-module-suse-manager-server/<productnumber>/x86_64
```

File: modules/installation-and-upgrade/pages/install-vm.adoc (hunk 4 above)
Comment (keichwa) [reply modules/installation-and-upgrade/pages/install-vm.adoc:121]: Was introduced with cherry-picking.  Reverted.  I'll let Ornela and Joseph decide whether we want to use smrproductnumber here (iirc, this was intended for SUSE Manager Retail).

File: .changelog [hunk 6]
```diff
@@ -4,9 +4,11 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- SUSE Manager 4.3 is based on SUSE Linux Enterprise 15 SP4; changed
+  installation procedure accordingly (bsc1213469)
 - Added comment about SCC subscription to Administration Guide
   (bsc#1211270)
-- Documented Debian 12 as a supported client OS in Client Configuration 
+- Documented Debian 12 as a supported client OS in Client Configuration
```
Comment (0rnela) [inline .changelog:11]:
```suggestion
- Added Debian 12 as a technology preview client in Client Configuration Guide
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

File: modules/installation-and-upgrade/pages/install-vm.adoc (hunk 4 above)
Comment (0rnela) [reply modules/installation-and-upgrade/pages/install-vm.adoc:121]: We have decided that his line stays as it was in all previous versions of the documentation.

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/installation-and-upgrade/pages/proxy-registration.adoc [hunk 7]
```diff
@@ -33,10 +33,10 @@ image::proxy-key.png[]
...
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-{sp-base}-Pool
+* SLE-Module-Basesystem15-{sp-base}-Updates
+* SLE-Module-Server-Applications15-{sp-base}-Pool
+* SLE-Module-Server-Applications15-{sp-base}-Updates
 * SLE-Module-SUSE-Manager-Proxy-4.3-Pool
 * SLE-Module-SUSE-Manager-Proxy-4.3-Updates
```
Comment (deneb-alpha) [inline modules/installation-and-upgrade/pages/proxy-registration.adoc:41]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Pool
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Updates
```

File: modules/quickstart/pages/quickstart-install-suma-proxy.adoc [hunk 8]
```diff
@@ -61,10 +61,10 @@ image::proxy-key.png[]
...
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-{sp-base}-Pool
+* SLE-Module-Basesystem15-{sp-base}-Updates
+* SLE-Module-Server-Applications15-{sp-base}-Pool
+* SLE-Module-Server-Applications15-{sp-base}-Updates
 * SLE-Module-SUSE-Manager-Proxy-4.3-Pool
 * SLE-Module-SUSE-Manager-Proxy-4.3-Updates
```
Comment (deneb-alpha) [inline modules/quickstart/pages/quickstart-install-suma-proxy.adoc:69]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Pool
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Updates
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
what about using `{productnumber}` instead of `4.3`.

will that attribute be rendered as `4.3` or as `"4.3"`

**Comment by jcayouette** [comment]
> what about using `{productnumber}` instead of `4.3`.
> 
> will that attribute be rendered as `4.3` or as `"4.3"`

It would be `4.3`. Quotes are excluded in YAML. 

However, if we go this route I would prefer to create an issue to track it and update this later. I am certain we have other places for this attribute. We should not be adjusting content like this on the fly during a release.

File: modules/installation-and-upgrade/pages/proxy-registration.adoc [hunk 9]
```diff
@@ -33,12 +33,12 @@ image::proxy-key.png[]
...
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
-* SLE-Module-SUSE-Manager-Proxy-4.3-Pool
-* SLE-Module-SUSE-Manager-Proxy-4.3-Updates
+* SLE-Module-Basesystem15-{sp-base}-Pool
+* SLE-Module-Basesystem15-{sp-base}-Updates
+* SLE-Module-Server-Applications15-{sp-base}-Pool
+* SLE-Module-Server-Applications15-{sp-base}-Updates
+* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Pool
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/proxy-registration.adoc:40]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-4.3-Pool
```

File: modules/installation-and-upgrade/pages/proxy-registration.adoc [hunk 10]
```diff
@@ -33,12 +33,12 @@ image::proxy-key.png[]
...
+* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Updates
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/proxy-registration.adoc:41]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-4.3-Updates
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

File: modules/installation-and-upgrade/pages/proxy-registration.adoc (hunk 9 above)
Comment (0rnela) [reply modules/installation-and-upgrade/pages/proxy-registration.adoc:40]: Reverting back as per @jcayouette comment (see below). I agree with his POV, we need to have a clean issue and PRs dealing with this.

File: modules/installation-and-upgrade/pages/proxy-registration.adoc (hunk 10 above)
Comment (0rnela) [reply modules/installation-and-upgrade/pages/proxy-registration.adoc:41]: Reverting back as per @jcayouette comment before. I agree with his POV, we need to have a clean issue and PRs dealing with this.

File: modules/quickstart/pages/quickstart-install-suma-proxy.adoc [hunk 11]
```diff
@@ -61,12 +61,12 @@ image::proxy-key.png[]
...
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
-* SLE-Module-SUSE-Manager-Proxy-4.3-Pool
-* SLE-Module-SUSE-Manager-Proxy-4.3-Updates
+* SLE-Module-Basesystem15-{sp-base}-Pool
+* SLE-Module-Basesystem15-{sp-base}-Updates
+* SLE-Module-Server-Applications15-{sp-base}-Pool
+* SLE-Module-Server-Applications15-{sp-base}-Updates
+* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Pool
```
Comment (0rnela) [inline modules/quickstart/pages/quickstart-install-suma-proxy.adoc:68]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-4.3-Pool
```

File: modules/quickstart/pages/quickstart-install-suma-proxy.adoc [hunk 12]
```diff
@@ -61,12 +61,12 @@ image::proxy-key.png[]
...
+* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Updates
```
Comment (0rnela) [inline modules/quickstart/pages/quickstart-install-suma-proxy.adoc:69]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-4.3-Updates
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

**Review by 0rnela — APPROVED** [review]

**Review by deneb-alpha — APPROVED** [review]
LGTM, thanks

**Pull request merged**